Packs/Phishing/Scripts/FindEmailCampaign/FindEmailCampaign.py

```diff
@@ -0,0 +1,440 @@
+            'preview', 'price', 'prize', 'profit', 'promise', 'promotion', 'purchase', 'pure', 'qualifies', 'qualify',
+            'quote', 'rates', 'receipt', 'record', 'recorded', 'recording', 'refund', 'request', 'requested',
+            'requires', 'reserve', 'reserves', 'review', 'risk', 'sales', 'satisfactin', 'satisfaction', 'satisfied',
+            'save', 'scam', 'security', 'sensitive', 'sex', 'share', 'shared', 'sharing', 'shipment', 'shipping',
+            'sir', 'spam', 'special', 'spend', 'spending', 'started', 'starting', 'stock', 'success', 'supplies',
+            'supply', 'suspended', 'temporarily', 'terms', 'trader', 'trading', 'traffic', 'transaction', 'transfer',
+            'trial', 'unlimited', 'unsecured', 'unsolicited', 'unsubscribe', 'update', 'ups', 'urgent', 'user', 'usps',
+            'valium', 'verification', 'verify', 'viagra', 'vicodin', 'videos', 'vids', 'viedo', 'virus', 'waiting',
+            'wallet', 'warranty', 'web', 'weight', 'win', 'winner', 'winning', 'wire', 'xanax']
+
+STATUS_DICT = {
+    0: "Pending",
+    1: "Active",
+    2: "Closed",
+    3: "Archive",
+}
+
+
+def return_outputs_custom(readable_output, outputs=None, tag=None):
+    return_entry = {
+        "Type": entryTypes["note"],
+        "HumanReadable": readable_output,
+        "ContentsFormat": formats['json'],
+        "Contents": outputs,
+        "EntryContext": outputs,
+    }
+    if tag is not None:
+        return_entry["Tags"] = ['campaign_{}'.format(tag)]
+    demisto.results(return_entry)
+
+
+def add_context_key(entry_context):
+    new_context = {}
+    for k, v in entry_context.items():
+        new_context['{}.{}'.format('EmailCampaign', k)] = v
+    return new_context
+
+
+def create_context_for_campaign_details(campaign_found=False, incidents_df=None):
+    if incidents_df is None:
+        incidents_context = []
+    else:
+        incident_id = demisto.incident()['id']
+        incident_df = incidents_df[['id', 'similarity', FROM_FIELD, FROM_DOMAIN_FIELD]]
+        incident_df = incident_df[incident_df['id'] != incident_id]
+        incident_df.rename({FROM_DOMAIN_FIELD: 'emailfromdomain'}, axis=1, inplace=True)
+        incidents_context = incident_df.fillna(1).to_dict(orient='records')
+    return {
+        'isCampaignFound': campaign_found,
+        'involvedIncidentsCount': len(incidents_df) if incidents_df is not None else 0,
+        'incidents': incidents_context
+    }
+
+
+def create_context_for_indicators(indicators_df=None):
+    if indicators_df is None:
+        indicators_context = []
+    else:
+        indicators_df.rename({'Value': 'value'}, axis=1, inplace=True)
+        indicators_df = indicators_df[['id', 'value']]
+        indicators_context = indicators_df.to_dict(orient='records')
+    return {'indicators': indicators_context}
+
+
+CONTEXT_FUNCTIONS = [create_context_for_campaign_details, create_context_for_indicators, ]
+
+
+def create_empty_context():
```

Comment:
usually in this case we can just not reutrn a context, or to return the campaign idenfity false context key. why we need all the keys?